google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -142,14 +143,24 @@ def _get_operator_definition(self, task_id, task_details):
     operator_param_values = Pipeline._get_operator_param_name_and_values(
         operator_classname, task_details)
     for (operator_param_name, operator_param_value) in sorted(operator_param_values.items()):
+      operator_param_value = self._resolve_airflow_macros(operator_param_value)
       param_format_string = Pipeline._get_param_format_string(
           operator_param_value)
       param_string = param_format_string.format(operator_param_name, operator_param_value)
-      param_string = param_string % self._airflow_macros
       full_param_string = full_param_string + param_string
 
     return '{0} = {1}({2}, dag=dag)\n'.format(task_id, operator_classname, full_param_string)
 
+  def _resolve_airflow_macros(self, operator_param_value):
+    if type(operator_param_value) is list:
```

Comment:
isinstance(operator_param_value, list) as is mostly done in pydatalab.

Reply:
Done.